net-libs/libslirp/patches/libslirp-4.7.0.patchset

```diff
@@ -0,0 +1,24 @@
+From 78faa99ec2886bdf8af8e8719980ed9868851ae4 Mon Sep 17 00:00:00 2001
```

Comment:
Could you upstream this patch?

Reply:
https://gitlab.freedesktop.org/slirp/libslirp/-/merge_requests/123
We'll see how this goes

Reply:
They've already merged it upstream :)